src/redux/reducer.js

```diff
@@ -683,8 +683,14 @@ export const reducer = createReducer(initialState, {
                             diagram.state = ViewState.MINIMIZED;
                         }
                     });
+                    const diagramToOpen = diagramStates[diagramToOpenIndex];
+
+                    // we push the diagram to the last position, so when we reopen it, it opens at the last position available.
```

Comment:
The comment should be updated to :
`// We open and push the SLD to the last position in the array so it is displayed at the right of the others`

and the `diagramToOpen.state = ViewState.OPENED;` should be on the next line, before the splice, to make the code easier to read.